src/test/java/core/basesyntax/service/impl/FruitTransactionParserImplTest.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitTransactionParser;
+import java.util.List;
+import org.junit.Test;
+
+public class FruitTransactionParserImplTest {
+    private static final FruitTransactionParser parser = new FruitTransactionParserImpl();
+    private static final int FIRST_TRANSACTION_INDEX = 0;
+
+    @Test
+    public void parseTransactionNotParseHeader_Ok() {
```

Comment:
parseTransaction_ok
 add with null data test

Reply:
I can write the test with null, but i don't think we need it in this case, as long as the list of strings comes from file, and null is impossible here